.github/workflows/2x-kubernetes-version-test.yaml

```diff
@@ -0,0 +1,128 @@
+name: 2.x Kubernetes Version Tests
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'next'
+
+jobs:
+  test-kubernetes-1-19-dbless:
```

Comment:
See how go-kong achieves multiversion testing without code duplication:

https://github.com/Kong/go-kong/blob/100d6b60b97942ccaa2c5c319fd72bad53bc9341/.github/workflows/integration-test.yaml#L9

I think that something similar should be doable here - for dbless/postgres as well as for k8s versions.

Reply:
Sounds good, I'll switch to using that strategy instead.

Reply:
Well... this does end up making the status checks a lot more verbose but since this wont run on every PR anyway I think that'll be just fine.

Resolved with 4b02baa6ede4b68a32706f260dd31dec9728f142 :+1: